scout/server/blueprints/variants/controllers.py

```diff
@@ -1060,84 +1058,172 @@ def generate(header, lines):
+
+    variant_line.append(get_fusion_exons(variant))
+
+    variant_gts = variant["samples"]  # list of coverage and gt calls for case samples
+    for individual in case_obj["individuals"]:
+        for variant_gt in variant_gts:
+            if individual["individual_id"] != variant_gt["sample_id"]:
+                continue
+            variant_line.append(variant_gt.get("read_depth", "N/A"))
+            variant_line.append(variant_gt.get("split_read", "N/A"))
+            variant_line.append(variant_gt.get("ffpm", "N/A"))
+
+    return variant_line
+
+
+def variant_export_lines_cancer(variant: dict) -> list:
+    """
+    Get cancer specific variant info to be exported. Returns a list to be merged into a string
+    in suitable export format.
+    """
+    variant_line = []
+
+    # Add cancer and normal VAFs
+    for sample in ["tumor", "normal"]:
+        allele = variant.get(sample)
+        if not allele:
+            variant_line.append("-")
+            continue
+        alt_freq = round(allele.get("alt_freq", 0), 4)
+        alt_depth = allele.get("alt_depth")
+        ref_depth = allele.get("ref_depth")
+
+        vaf_sample = f"{alt_freq} ({alt_depth}|{ref_depth})"
+        variant_line.append(vaf_sample)
+
+    # ADD eventual COSMIC ID
+    cosmic_ids = variant.get("cosmic_ids") or ["-"]
+    variant_line.append(" | ".join(cosmic_ids))
+
+    return variant_line
+
+
+def variant_export_lines_rare(variant: dict, case_obj: dict) -> list:
+    """
+    Get generic rare disease variant info to be exported. Returns a list to be merged into a string
+    in suitable export format.
+    """
+    variant_line = []
+
+    variant_gts = variant["samples"]  # list of coverage and gt calls for case samples
+    for individual in case_obj["individuals"]:
+        for variant_gt in variant_gts:
+            if individual["individual_id"] != variant_gt["sample_id"]:
+                continue
+
+            variant_line.append(variant_gt["genotype_call"])
+            # gather coverage info
+            variant_line.append(variant_gt["allele_depths"][0])  # AD reference
+            variant_line.append(variant_gt["allele_depths"][1])  # AD alternate
+            # gather genotype quality info
+            variant_line.append(variant_gt["genotype_quality"])
+
+    return variant_line
+
+
+def variant_export_lines(
+    store: MongoAdapter, case_obj: dict, variants_query: CursorType, category: Optional[str] = None
+):
     """Get variants info to be exported to file, one list (line) per variant.
     Args:
         store(scout.adapter.MongoAdapter)
         case_obj(scout.models.Case)
         variants_query: a list of variant objects, each one is a dictionary
     Returns:
         export_variants: a list of strings. Each string  of the list corresponding to the fields
-                         of a variant to be exported to file, separated by comma
+                         of a variant to be exported to file, separated by comma.
```

Comment:
Remove Args and Returns and instead add return type to the function?

Reply:
And here we know it is a `List[str]`!